logos/logos-ui/src/app/features/statistics/statistics.scss

```
@@ -149,6 +149,10 @@
   align-items: flex-end;
   height: 28px;
   gap: 2px;
+  // The bars are fixed-width and cannot compress, so in a card too narrow
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)